shenyu-sync-data-center/shenyu-sync-data-zookeeper/pom.xml

```diff
@@ -25,6 +25,11 @@
     <modelVersion>4.0.0</modelVersion>
     <artifactId>shenyu-sync-data-zookeeper</artifactId>
 
+    <properties>
+        <curator.version>4.0.1</curator.version>
```

Comment:
move this into parent pom.xml

Reply:
The version in parent pom is 5.2.1. There're many places use curator like mentioned in the ISSUE #3556 . I plan to fix them step by step and after fixed all of them. I'll move it in root pom and clean all the child pom.

Reply:
Or I can do them in one PR.